src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -244,6 +245,12 @@ If a profile is not specified, it runs the extension using a new temporary profi
 
 Environment variable: `$WEB_EXT_FIREFOX_PROFILE`
 
+#### `--profile-create-if-missing`
+
+This option is to be used with `--firefox-profile` or `--chromium-profile`. With this option, the profile option provided will be treated as a directory path and if the directory does not exist, it will be created.
+
+Environment variable: `$WEB_EXT_PROFILE_CREATE_IF_MISSING`
+
```

Comment:
Nit, how about tweaking this description a bit to make it consistent with the --keep-profile-changes option that follows? e.g. something like:

```
With this option, the profile directory (specified by the `--firefox-profile` or `--chromium-profile` options) will be created if it does not exist yet. 

::: note alert
When this option is specified the `--firefox-profile` option is always treated as a directory path.
:::

Environment variable: `$WEB_EXT_PROFILE_CREATE_IF_MISSING`
```

Reply:
Definitely! This is more concise and clear.